varcode/effects/effect_classes.py

```diff
@@ -617,14 +617,24 @@ def __init__(
             self,
             variant,
             transcript,
-            extended_protein_sequence):
-        aa_mutation_start_offset = len(transcript.protein_sequence)
+            extended_protein_sequence,
```

Comment:
To keep it in syc with the other classes, perhaps having aa_ref/aa_alt in that order would make more sense.